libs/api/domains/vehicles/src/lib/api-domains-vehicles.service.ts

```diff
@@ -142,10 +143,14 @@ export class VehiclesService {
   }
 
   async getPublicVehicleSearch(search: string) {
-    const data = await this.publicVehiclesApi.publicVehicleSearchGet({
-      search,
-    })
-    return data
+    try {
+      const data = await this.publicVehiclesApi.publicVehicleSearchGet({
+        search,
+      })
+      return data
+    } catch (error) {
+      handle404(error)
+    }
```

Comment:
Nice 🙌  should probably just change to this though:

```suggestion
    return await this.publicVehiclesApi
      .publicVehicleSearchGet({
        search,
      })
      .catch(handle404)
```

So it will return other errors

Reply:
Done